tests/cephadm/test_package_validation.py

```diff
@@ -89,6 +89,16 @@ def validate_packages(node, pkgs):
         # Compare installed package version
         compare_packages(node, pkgs)
 
+        # Check client additional package "ceph-diff-sorted" and "rgw-orphan-list"
+        if "ceph-common" in pkgs:
+            out = node.exec_command(
+                sudo=True,
+                cmd="ls -l /usr/bin/rgw-or* /usr/bin/ceph-diff*",
```

Comment:
Can't we use - https://github.com/MohitBis/cephci/blob/67beaa5d54dd8ce95b49d3e32da4b54b83b63a3b/cli/utilities/packages.py#L276 ?

Reply:
@vamahaja 
We can't use mentioned lib since only way to check the package it mentioned command as updated in TC and bug
Please refer TC: 83575601

Reply:
Ack